packages/handlers/odata/yaml-config.graphql

```diff
@@ -0,0 +1,16 @@
+extend type Handler {
+  oData: ODataHandler
+}
+
+type ODataHandler {
+  baseUrl: String!
+  operationHeaders: JSON
```

Comment:
Interpolation works on this one? Because we might need it. To allow changing access tokens.

Reply:
Yes it works for both headers and URLs